examples/file_browser.html

```diff
@@ -0,0 +1,119 @@
+<!--
+
+   Copyright (c) 2020, the Regular Table Authors.
+
+   This file is part of the Regular Table library, distributed under the terms of
+   the Apache License 2.0.  The full license can be found in the LICENSE file.
+
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <script src="../umd/regular-table.js"></script>
+    <link rel='stylesheet' href="../css/material.css">
+</head>
+
+<body>
+
+    <regular-table></regular-table>
+
+    <script>
+
+        const NUM_ROWS = 100;
+        const NUM_DIR = 10;
+        const NUM_FILE = NUM_ROWS - NUM_DIR;
```

Comment:
Unused `NUM_FILE`

Reply:
thanks for the spot! hangover from earlier code

Reply:
done!